### PR TITLE
Chore: Cleanup Mixed British/American Spelling to American English

### PR DESCRIPTION
**Pull request by hooksta4**
While American English doesn't have the corner on the English language, there were multiple British spellings and American spellings in the same file, in some instances. There were many more American spellings of these words than British so I went with the American way of spelling. 

Standardized spelling of: 
initialisation to initialization.
colour to color
behaviour to behavior
Added a migration to rename char_chocobos column colours to color

**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Standardizes the spelling of 
Initialization, Colour, Behaviour 

## Steps to test these changes
Build server and no longer will see initialisation

### CODE REVIEW

**Review by Xaver-DaRed — APPROVED** [review]

**Comment by zach2good** [comment]
_Sad English noises_

For some reason I was thinking about doing this last week, strange.

I'm not opposed to this change, but you need to do it correctly. As you can imagine, the `ext` folder is external code, not modified or maintained by us, that stands the risk of being copy-pasted over when/if we update it. So there's no point in you applying these changes there.

There's also more English vs American spellings in the codebase which you should change if you want to standardise.

image

image

image

https://libguides.hull.ac.uk/grammar/englishnotamerican

If you're only interested in changing this one visible log message, just change it to `init` and then neither variant of English is used.

**Comment by cocosolos** [comment]
Several uses of **initialise** as well, notably luautils OnZoneInitialise calls Lua onZoneInitialize.

**Comment by zach2good** [comment]
In the case of Initialise/Initialize, I'd probably just slap it all as `Init` and be done with it forever

**Comment by hooksta4** [comment]
> There's also more English vs American spellings in the codebase which you should change if you want to standardise.

Touché  sir... I am not interested in having a english english comparison but rather just standardiZing one way or the other. The Z seems to have a lot more usage than the S. And for the color - well that isn't a debate. You folks across the pond spell it wrong ;)

**Comment by hooksta4** [comment]
>------ Build All started: Project: server, Configuration: x64-Debug ------
  [1/50] Building CXX object src\login\CMakeFiles\xi_connect.dir\__\common\version.cpp.obj
  [2/50] Building CXX object src\map\CMakeFiles\xi_map.dir\__\common\version.cpp.obj
  [3/50] Building CXX object src\map\CMakeFiles\xi_map.dir\latent_effect.cpp.obj
  [4/50] Building CXX object src\map\CMakeFiles\xi_map.dir\notoriety_container.cpp.obj
  [5/50] Building CXX object src\map\CMakeFiles\xi_map.dir\recast_container.cpp.obj
  [6/50] Building CXX object src\map\CMakeFiles\xi_map.dir\guild.cpp.obj
  [7/50] Building CXX object src\map\CMakeFiles\xi_map.dir\merit.cpp.obj
  [8/50] Building CXX object src\map\CMakeFiles\xi_map.dir\job_points.cpp.obj
  [9/50] Building CXX object src\map\CMakeFiles\xi_map.dir\daily_system.cpp.obj
  [10/50] Building CXX object src\map\CMakeFiles\xi_map.dir\mob_spell_container.cpp.obj
  [11/50] Building CXX object src\map\CMakeFiles\xi_map.dir\packet_guard.cpp.obj
  [12/50] Building CXX object src\map\CMakeFiles\xi_map.dir\transport.cpp.obj
  [13/50] Building CXX object src\map\CMakeFiles\xi_map.dir\party.cpp.obj
  [14/50] Building CXX object src\map\CMakeFiles\xi_map.dir\instance.cpp.obj
  [15/50] Building CXX object src\map\CMakeFiles\xi_map.dir\battlefield_handler.cpp.obj
  [16/50] Building CXX object src\map\CMakeFiles\xi_map.dir\enmity_container.cpp.obj
  [17/50] Building CXX object src\map\CMakeFiles\xi_map.dir\linkshell.cpp.obj
  [18/50] Building CXX object src\map\CMakeFiles\xi_map.dir\latent_effect_container.cpp.obj
  [19/50] Building CXX object src\map\CMakeFiles\xi_map.dir\mob_spell_list.cpp.obj
  [20/50] Building CXX object src\map\CMakeFiles\xi_map.dir\conquest_system.cpp.obj
  [21/50] Building CXX object src\map\CMakeFiles\xi_map.dir\monstrosity.cpp.obj
  [22/50] Building CXX object src\map\CMakeFiles\xi_map.dir\instance_loader.cpp.obj
  [23/50] Building CXX object src\map\CMakeFiles\xi_map.dir\spell.cpp.obj
  [24/50] Building CXX object src\map\CMakeFiles\xi_map.dir\message.cpp.obj
  [25/50] Building CXX object src\map\CMakeFiles\xi_map.dir\roe.cpp.obj
  [26/50] Building CXX object src\search\CMakeFiles\xi_search.dir\__\common\version.cpp.obj
  [27/50] Building CXX object src\map\CMakeFiles\xi_map.dir\unitychat.cpp.obj
  [28/50] Building CXX object src\world\CMakeFiles\xi_world.dir\__\common\version.cpp.obj
  [29/50] Building CXX object src\map\CMakeFiles\xi_map.dir\timetriggers.cpp.obj
  [30/50] Building CXX object src\map\CMakeFiles\xi_map.dir\time_server.cpp.obj
  [31/50] Building CXX object src\map\CMakeFiles\xi_map.dir\map.cpp.obj
  [32/50] Building CXX object src\map\CMakeFiles\xi_map.dir\trait.cpp.obj
  [33/50] Building CXX object src\map\CMakeFiles\xi_map.dir\treasure_pool.cpp.obj
  [34/50] Linking CXX executable "C:\Users\carlh\OneDrive\Documents\FFXI STUFF\Repo\server\xi_connect.exe"
  [35/50] Building CXX object src\map\CMakeFiles\xi_map.dir\zone_instance.cpp.obj
  [36/50] Building CXX object src\map\CMakeFiles\xi_map.dir\status_effect_container.cpp.obj
  [37/50] Building CXX object src\map\CMakeFiles\xi_map.dir\packet_system.cpp.obj
  [38/50] Building CXX object src\map\CMakeFiles\xi_map.dir\zone_entities.cpp.obj
  [39/50] Building CXX object src\map\CMakeFiles\xi_map.dir\zone.cpp.obj
  [40/50] Building CXX object src\world\CMakeFiles\xi_world.dir\conquest_system.cpp.obj
  [41/50] Building CXX object src\search\CMakeFiles\xi_search.dir\__\common\socket.cpp.obj
  [42/50] Building CXX object src\world\CMakeFiles\xi_world.dir\message_server.cpp.obj
  [43/50] Building CXX object src\search\CMakeFiles\xi_search.dir\__\common\application.cpp.obj
  [44/50] Building CXX object src\world\CMakeFiles\xi_world.dir\__\common\application.cpp.obj
  [45/50] Building CXX object src\world\CMakeFiles\xi_world.dir\time_server.cpp.obj
  [46/50] Building CXX object src\world\CMakeFiles\xi_world.dir\main.cpp.obj
  [47/50] Building CXX object src\world\CMakeFiles\xi_world.dir\world_server.cpp.obj
  [48/50] Linking CXX executable "C:\Users\carlh\OneDrive\Documents\FFXI STUFF\Repo\server\xi_search.exe"
  [49/50] Linking CXX executable "C:\Users\carlh\OneDrive\Documents\FFXI STUFF\Repo\server\xi_world.exe"
  [50/50] Linking CXX executable "C:\Users\carlh\OneDrive\Documents\FFXI STUFF\Repo\server\xi_map.exe"

Build All succeeded.

**Comment by zach2good** [comment]
@hooksta4 I deleted your comment because it revealed your name and I've never seen you do that on your own.

**Comment by hooksta4** [comment]
> @hooksta4 I deleted your comment because it revealed your name and I've never seen you do that on your own.

Good call - thanks - I accidently kept a few of the /ext will remove those shortly.

**Comment by zach2good** [comment]
British English to American English. America doesn't have a monopoly on 'English'. Thanks.

File: tools/migrations/043_modify_choco_colour_column.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+def migration_name():
+    return "Rename colour column to color in char_chocobos"
+
+
+def check_preconditions(cur):
+    return
+
+
+def needs_to_run(cur):
+    cur.execute("SHOW COLUMNS FROM char_chocobos ;")
```
Comment (cocosolos) [inline tools/migrations/043_modify_choco_colour_column.py]: This looks incomplete, should check if a colour column exists.

Comment (hooksta4) [reply tools/migrations/043_modify_choco_colour_column.py]: Colour or color? 

```lua
def needs_to_run(cur):
    # Ensure column doesn't already exist.
    cur.execute("SHOW COLUMNS FROM char_chocobos LIKE 'color';")

    row = cur.fetchone()
    if row:
        return False

    return True
    ```

Comment (cocosolos) [reply tools/migrations/043_modify_choco_colour_column.py]: Don't think it matters, either way should be fine.

**Pull request merged**